pkg/staticnetworkconfig/generator.go

```diff
@@ -186,23 +189,14 @@ func (s *StaticNetworkConfigGenerator) validateInterfaceNamesExistence(macInterf
 		switch interfaceType.String() {
 		case "802-3-ethernet", "ethernet":
```

Comment:
This check is not needed if we  matching only `enXXX` it will be always `ethernet`

Reply:
I believe they're aliases of each other.